### ISSUE TITLE
Research framework migration paths

### ISSUE THREAD

**Issue by slaweet** [LiskHQ/lisk-nano#311]
We want to migrate Angular 1 to something newer (Angular 2, React, ???). Let's research pros and cons of the options we have and how can the migration be executed.

**Comment by alepop** [comment]
Angular is good at the start of a project, but in my opinion is become harder when the code base is growing. I think that the stack like Typescript + React + Redux/MobX (i use it on my everyday work) is good. Because in any time you can change any library for another.

**Comment by alepop** [comment]
And I recommend you to look at the [Elm](http://elm-lang.org) lang!!! Redux was inspired by this lang.

**Comment by reyraa** [comment]
@alepop  We'll have to consider the cost of migration in case of time. Although I personally prefer react, but we need a throughout investigation to make sure about all aspect of it. also considering the future plans.

**Comment by slaweet** [comment]
After the research and discussion we did, we concluded to go with React and Redux. Compared to Angular 2, React will give us more flexibility in choosing/swapping tools/plugins for our performance and other needs.

Also in future, we would like to use RactNative, which appears to be more mature than Angular 2 alternative - NativeScript.

Closing this issue as there are other ones to do the migration:
Set up boilerplate for React migration #342
Migrate to Angular 2 or React or other contemporary framework #279
